docs/changes/12179.other.rst

```diff
@@ -0,0 +1,3 @@
+When ``astropy`` raises an ``OSError`` because a file it was told to write
+already exists then the error message now suggests the use of the
```

Comment:
```suggestion
already exists, the error message now suggests the use of the
```

Reply:
Looks like I missed this when I last pushed. I'll include it in the next commit.